mage_integrations/mage_integrations/sources/constants.py

```diff
@@ -4,21 +4,26 @@
 COLUMN_TYPE_ARRAY = 'array'
 COLUMN_TYPE_BINARY = 'binary'
 COLUMN_TYPE_BOOLEAN = 'boolean'
+COLUMN_TYPE_FLOAT = 'float'
 COLUMN_TYPE_INTEGER = 'integer'
 COLUMN_TYPE_NULL = 'null'
 COLUMN_TYPE_NUMBER = 'number'
 COLUMN_TYPE_OBJECT = 'object'
 COLUMN_TYPE_STRING = 'string'
+COLUMN_TYPE_VARIANT = 'variant'
```

Comment:
No need to add new column types.

Reply:
@wangxiaoyou1993 there is an error message related to `variant` type: 
```
 expecting VARIANT but got VARCHAR(16777216) for column variant1.
```
How to resolve this issue without introducing a new data type?

Reply:
where do you get the error message?

Reply:
I mean this [commit](https://github.com/mage-ai/mage-ai/pull/2896/commits/2ba5a33dae8f884190b9ea42dbee68faf7c90d36) is not needed.

Reply:
For a field with the type `object` initially, is it Ok to map it to `variant` in the destination?

Reply:
Yes, the `object` type is already mapped to `variant` in destination

Reply:
For an `array` field, the Snowflake source generates a `varchar` type at present, is it Ok to keep the `varchar` type instead of maintaining the `array` type?

Reply:
The current Snowflake source uses the default data type conversion, which didn't handle the array type somehow: https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/sources/sql/base.py#L106

Reply:
ok. then just keep it as it is.